src/main/resources/config/application-prod.yml

```diff
@@ -59,6 +59,9 @@ spring:
             hibernate.cache.use_query_cache: false
             hibernate.generate_statistics: false
             hibernate.cache.use_minimal_puts: true
+            hibernate.dialect: io.github.jhipster.domain.util.FixedPostgreSQL82Dialect
```

Comment:
this is unnecessary properties, this information already defined in the section

```
jpa:
        database-platform: 
```

Reply:
done